src/material/icon/icon.scss

```diff
@@ -59,6 +59,26 @@ mat-icon {
 
     .mat-icon-button .mat-icon {
       margin: auto;
+      overflow: visible;
     }
   }
 }
+
+// Makes button icon overflow visible to fix b/250063405
+.mat-mdc-icon-button .mat-icon,
+.mdc-button .mat-icon,
+.mat-mdc-fab-base .mat-icon,
+.mat-toolbar .mat-icon,
+.mat-mdc-tooltip-trigger .mat-icon,
+.mat-mdc-tab .mat-icon,
+.mat-mdc-form-field .mat-icon,
+.mat-grid-tile .mat-icon,
+.mat-mdc-list-item .mat-icon,
+.mat-mdc-list-option .mat-icon,
+.mat-icon:has(div[style*='letter-spacing:']):has(div[style*='word-spacing:']) {
+  overflow: visible;
+  // Apply styles to avoid text spacing
+  letter-spacing: 0 !important;
```

Comment:
Why do we need to add the `!important`? We can apply these as regular styles on `mat-icon`. If the app happens to be overriding them, it's on them to remove the overrides.

Reply:
Hi @crisbeto, I was attempting to have the specificity+`!important` override the injected stylesheet and/or user agent stylesheets which Parashu and Text Spacing Editor (Chrome extensions) applied styles since it looks like they both target `*` with `letter-spacing`, `word-spacing`, and `line-height` with `!important` flags. See [Parashu override screenshot](https://screenshot.googleplex.com/uESukzBbcmt7cFU) and [Text Space Editor override screenshot](https://screenshot.googleplex.com/7k2vMA9RfaHKujs) for examples.

Regardless, it looks like if there are external/user agent stylesheets using `!important` that those are unable to be overridden according to [this article](https://www.stefanjudis.com/today-i-learned/you-cant-override-important-user-agent-css-declarations/) I found.

I've gone ahead and removed the targets to inline-styles as well as the usage of `!important` based on your recommendations.

Let me know if I should change anything else.